_format/1.1/index.md

```diff
@@ -319,6 +319,12 @@ The top-level [links object][links] **MAY** contain the following members:
   Schema) for the current document.
 * [pagination] links for the primary data.
 
+> Note: The `self` link should contain the query parameters provided by the client
```

Comment:
I'd suggest changing

``The `self` link``

to something like

``The `self` link in the top-level `links` object``

because it's already very easy to lose the specific context of a requirement based on headers even just a few paragraphs away.

Also, ``should`` should be ``**MUST**``, because it's a requirement that the self link reflects the URL that generated the document.

Reply:
```suggestion
> Note: The `self` link in the top-level `links` object must contain the query parameters provided by the client
```

Reply:
So why `must` instead of `**MUST**`?  The later is specifically how a compliance point is indicated throughout the specification.

Reply:
I don't see this note as part of the specification. It is an additional explanation provided with the spec. Due to this different intent it should use normal language rather than normative specification terms. This is inline with existing notes in the spec.